build/KoreBuild.ps1

```diff
@@ -123,4 +123,20 @@ if (!(Test-Path $msbuildArtifactsDir))
 $msBuildArguments | Out-File -Encoding ASCII -FilePath $msBuildResponseFile
 
 __exec dotnet restore /p:PreflightRestore=true "$makeFileProj"
-__exec dotnet msbuild `@"$msBuildResponseFile"
+
+try {
+    __exec dotnet msbuild `@"$msBuildResponseFile"
+} finally {
+    # If the structured logger is enabled, replay the binary log to a structured log and remove the binary log
```

Comment:
nit: remove the part `and remove the binary log ` as we want to be able to replay to text file if needed?